vendor/sigs.k8s.io/kustomize/api/resource/resource.go

```diff
@@ -239,15 +240,6 @@ func (r *Resource) ReferencesEqual(other *Resource) bool {
 	return len(setSelf) == len(setOther)
 }
 
-func copyStringSlice(s []string) []string {
```

Comment:
you should not edit code from vendor directly

Reply:
Removed the change, 😄 accidentally did to much